media-sound/ladish/ladish-1.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit python-single-r1 waf-utils
+
+DESCRIPTION="LADI Session Handler - a session management system for JACK applications"
+HOMEPAGE="https://ladish.org"
+
+SRC_URI="https://dl.ladish.org/ladish/${P}-g36c489e4.tar.bz2"
+S="${WORKDIR}/${P}-g36c489e4"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
Hmm, was this tested on all this arches? Otherwise i would stick to those arches for which you build it for.

Reply:
It is a 10+ year old software that I've personally built in one time or another on all these four arches. The hardware I use daily is arm64, but marking this stable for arm64 is against the current Gentoo rules (AFAIK).

Reply:
Oh, ok.. well then it should be fine. :)
BTW, i didn't mean to mark it stable - just to remove arches which weren't tested. But if it runs on all these arches, it's fine to keep it.

Reply:
We add new arches via keyword reqs - not randomly adding them in bumps. See https://devmanual.gentoo.org/keywording/index.html.